app/views/concepts/_show.html.haml

```diff
@@ -3,72 +3,73 @@
     = render Display::AlertComponent.new do
       = t('concepts.use_jump_to')
   - else
-    = render TabsContainerComponent.new(type:'outline') do |c|
+    %div{'data-controller': 'concepts-json', 'data-concepts-json-class-url-value': "#{@ontology.id}/classes/#{escape(@concept.id)}", 'data-concepts-json-api-key-value': get_apikey}
+      #concept_tabs_container{'data-action': 'click->concepts-json#update'}
```

Comment:
remove ` #concept_tabs_container` and put the action directly in the parent `div`